andela/ah-robotics-frontend
# #162949032 Setup react router version 4 (React-router-dom)

#### What does this PR do?

Enables routing with react router version 4 

#### Description of Task to be completed?

- [x] Create a components folder
- [x] Create a routes folder
- [x] Add home, log in and error components and views
- [x] route the pages


#### How should this be manually tested?

1. Clone this repository on your local machine using 

     `git clone https://github.com/andela/ah-robotics-frontend.git`
2. Change your working directory to the project's directory using 
     
     `cd ah-robotics-frontend`
3. Change to the specific branch `ch-react-router-v4-162949032` using 

     `git checkout ch-react-router-v4-162949032`

4. Run `npm install` to install all the modules listed in the `package.json` file.

5. Run `npm start` command to start the application.

6. The application should start successfully displaying a page as shown below.

<img width="1280" alt="screen shot 2019-02-07 at 18 04 43" src="https://user-images.githubusercontent.com/14871749/52420321-eb5a3f80-2b02-11e9-808c-6f4e47984777.png">


7. On the top navigation bar Click on the `Log In` link to go to the Login page below:

<img width="1280" alt="screen shot 2019-02-07 at 18 04 22" src="https://user-images.githubusercontent.com/14871749/52420456-2a889080-2b03-11e9-908c-20dfc9b188dc.png">

8. The rest of the links will display the image below because they are not available:

<img width="1280" alt="screen shot 2019-02-07 at 18 09 31" src="https://user-images.githubusercontent.com/14871749/52420661-90751800-2b03-11e9-93a6-6709977c24e1.png">

#### What are the relevant pivotal tracker stories?

[#162949032](https://www.pivotaltracker.com/story/show/162949032)

[Finishes #162949032]
